src/domain/tradingview/utils.ts

```diff
@@ -0,0 +1,13 @@
+export const supportedResolutions = { 5: "5m", 15: "15m", 60: "1h", 240: "4h", "1D": "1d" };
```

Comment:
A little about consistency:
1. Maybe call it SUPPORTED_RESOLUTIONS like other constants?
2. It looks like some settings that can be changed, maybe put it in config/tradingView.ts?

Reply:
Done!